kitsune/sumo/static/sumo/js/legacy_login_toggle.js

```diff
@@ -1,13 +0,0 @@
-(function() {
```

Comment:
This file is still referenced in styleguide/kss-config.json

Reply:
Nice catch. I will remove it from there too.